tests/test_utils.py

```diff
@@ -47,3 +53,20 @@ def test_build_localized_fieldname(self):
         self.assertEqual(build_localized_fieldname("category__name", "nl"), "category__name_nl")
         self.assertEqual(build_localized_fieldname("title", "id"), "title_ind")
         self.assertEqual(build_localized_fieldname("title", "en-US"), "title_en_US")
+
+    def test_get_model_field(self):
```

Comment:
@jieter is het ook een idee om wat tests voor modelform mee te nemen? dan weten we ook of de get_field method vanuit modelform ook goed interacteerd met de modeltrans methoden?